pages/index.vue

```diff
@@ -29,4 +35,15 @@ const { data: packageInfo } = await useAsyncData<PackageInfo>(() =>
 );
 
 const qiskitVersion = packageInfo.value?.info.version ?? "";
+
+useSchemaOrg([
```

Comment:
If I'm not mistaken, this should go in the [default.vue layout](https://github.com/Qiskit/qiskit.org/blob/schema-espe/layouts/default.vue) ([according to this sentence from the nuxt-schema-org docs](https://unhead-schema-org.harlanzw.com/integrations/nuxt/module#:~:text=This%20should%20be,Component%20API)).

Something to consider: 
Should we also be adding this to the `default-max.vue`, and the `metal.vue` layouts as well?

Reply:
That is the recommended behavior, but as we're using default.vue as the header default, and schema.org usually goes inside the body tag,  for me it made more sense to include it there. 
Also, it only needs to be added in one place in the whole website.

Reply:
Thanks for the explanation

Reply:
I think Randy is right. By adding the `useSchemaOrg` on this page, we will only have the schema on this page. For having it on all the pages, we should add it or in the default layout or on the `app.vue` file (but we don't have it on our configuration).

You can easily check that it's not working as expected by looking at the `Elements` tab on the dev tools and looking for a line like this:

```
<script type="application/ld+json" id="schema-org-graph" ...>...</script>
```

You will notice that it's only on the main page and not on the rest

Reply:
per offline conversations, it has been decided to be kept on index so it shows only on the main page. 👍